CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Biconomy Projects 🚀
+
+First off, thank you for considering contributing to Biconomy! We truly appreciate your effort and contributions from the community are what make Biconomy awesome. 🙌
+
+Your contributions are valued and will help in driving the decentralized web forward. 
+
+> If you're passionate about our mission but can't contribute directly, there are other ways to support:
+> - ⭐ Star our projects on GitHub
+> - 🐦 [Tweet about Biconomy](https://twitter.com/biconomy/)
+> - 📌 Reference Biconomy in your project's readme
+> - 🗣️ Share about Biconomy at meetups or with peers
+
+## Table of Contents
+
+- [Contributing to Biconomy Projects 🚀](#contributing-to-biconomy-projects-)
+  - [Table of Contents](#table-of-contents)
+  - [Have a Question?](#have-a-question)
+  - [Ready to Contribute?](#ready-to-contribute)
+    - [Legal Notice 📜](#legal-notice-)
+    - [Reporting Bugs 🐛](#reporting-bugs-)
+    - [Suggesting Enhancements 💡](#suggesting-enhancements-)
+    - [First Time Contributing? 🌱](#first-time-contributing-)
+    - [Improving Documentation 📚](#improving-documentation-)
+  - [Style Guides 🎨](#style-guides-)
+    - [Commit Messages 📝](#commit-messages-)
+  - [Join Biconomy's Team! 🚀](#join-biconomys-team-)
+
+## Have a Question?
+
+Before reaching out, please ensure you've gone through our [Documentation](https://docs.biconomy.io/). If you still have questions:
+
+1. Search for existing [Issues](https://github.com/bcnmy/scw-contracts/issues) that might answer your question.
+2. Check out our [Forum](https://forum.biconomy.io/).
+3. Join our [Discord](https://discord.com/invite/biconomy) or [Telegram](https://t.me/biconomy) communities.
+
+If you still need assistance, feel free to open an [Issue](https://github.com/bcnmy/scw-contracts/issues/new) with your question.
+
+## Ready to Contribute?
+
+### Legal Notice 📜
+By contributing, you agree that you've authored your contribution and that it can be provided under the project's license.
+
+### Reporting Bugs 🐛
+
+Before submitting a bug report, ensure you're using the latest version and that you've read our [documentation](https://docs.biconomy.io/). If you've identified a bug that hasn't been reported, open a new [Issue](https://github.com/bcnmy/scw-contracts/issues/new) detailing the bug.
+
+### Suggesting Enhancements 💡
+
+Have a feature in mind? First, ensure it aligns with Biconomy's mission and hasn't been suggested before. Then, open an [Issue](https://github.com/bcnmy/scw-contracts/issues/new) to discuss your enhancement.
+
+### First Time Contributing? 🌱
+
+Welcome! We're thrilled to have you. If you're unsure where to start, look for issues labeled `good first issue`.
+
+### Improving Documentation 📚
+
+Good documentation is key! If you spot areas for improvement or errors in our documentation, we'd love your input.
```

Comment:
We can mention that if you need to suggest any changes in documentation, feel free to raise a PR on this repository https://github.com/bcnmy/docs